packages/devextreme/js/__internal/ui/toast/m_hide_toasts.ts

```diff
@@ -1,25 +1,37 @@
 import $ from '@js/core/renderer';
+import type Toast from '@ts/ui/toast/m_toast';
+import { TOAST_CLASS } from '@ts/ui/toast/m_toast';
 
-const TOAST_CLASS = 'dx-toast';
-
-function hideAllToasts(container): void {
+function hideAllToasts(container: Element): void {
```

Comment:
```suggestion
function hideAllToasts(container?: dxElementWrapper | Element): void {
  const toasts = $(`.${TOAST_CLASS}`).toArray();
  if (!container) {
    toasts.forEach((toast) => {
      Toast.getInstance(toast)?.hide();
    });
    return;
  }
```

let's refactor it this way. also i see it will break one test, but it's ok to update the test (pass jq el instead of htmlElement), minor change in behavior (check on undefined instead of arguments length) is ok, since function arg is not public and we have never shown this possibility to users